include/ibamr/MarangoniSurfaceTensionForceFunction.h

```diff
@@ -191,7 +191,7 @@ class MarangoniSurfaceTensionForceFunction : public SurfaceTensionForceFunction
     /*!
      * Marangoni coefficient.
      */
-    double d_marangoni_coefficient = std::numeric_limits<double>::signaling_NaN();
+    double d_marangoni_coefficient = 0.0;
```

Comment:
Is 0.0 a reasonable default value? If not we should keep this as the invalid value to be sure it is set later on.

Reply:
Thank you for pointing this out. Now  I initialize with std::numeric_limits<double>::signaling_NaN().